app/src/main/res/values/ember_fonts.xml

```diff
@@ -2,12 +2,12 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <!-- This file aliases Ember fonts, similar to Android Components
     https://github.com/mozilla-mobile/android-components/blob/master/components/ui/fonts/src/main/res/values/roboto_fonts.xml -->
-    <string name="font_ember_light">amazon-ember-display-light</string>
-    <string name="font_ember_regular">amazon-ember-display-regular</string>
-    <string name="font_ember_medium">amazon-ember-display-medium</string>
-    <string name="font_ember_bold">amazon-ember-display-bold</string>
-    <string name="font_ember_heavy">amazon-ember-display-heavy</string>
-</resources>
\ No newline at end of file
+    <string name="font_ember_light" translatable="false" tools:keep="@string/font_ember_light">amazon-ember-display-light</string>
+    <string name="font_ember_regular" translatable="false" tools:keep="@string/font_ember_regular">amazon-ember-display-regular</string>
+    <string name="font_ember_medium" translatable="false" tools:keep="@string/font_ember_medium">amazon-ember-display-medium</string>
+    <string name="font_ember_bold" translatable="false" tools:keep="@string/font_ember_bold">amazon-ember-display-bold</string>
+    <string name="font_ember_heavy" translatable="false" tools:keep="@string/font_ember_heavy">amazon-ember-display-heavy</string>
```

Comment:
The commit message says this suppresses 'unused', but I don't see that in the code.  Was it overlooked here?

Reply:
`tools:keep` is the way to suppress strings in resource files.